src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -2675,8 +2683,57 @@ public VirtualizingPanelSmoothScroller(VirtualizingPanelLayout layout, RecyclerV
 				_state = state;
 			}
 
+			public override int CalculateDtToFit(int viewStart, int viewEnd, int boxStart, int boxEnd, int snapPreference)
+			{
+				return base.CalculateDtToFit(viewStart, viewEnd, boxStart, boxEnd, -1);
+			}
+
+			public override int CalculateDyToMakeVisible(View view, int snapPreference)
+			{
+				return base.CalculateDyToMakeVisible(view, snapPreference);
+			}
```

Comment:
This override looks redundant.

Reply:
Thanks! That was not supposed to be pushed.